docker/py3-native/Dockerfile

```diff
@@ -0,0 +1,103 @@
+FROM demisto/py3-tools-ubi:1.0.0.40055
```

Comment:
please add the last updated line like we have in other dockerfiles

Reply:
we don't have this in all other docker files, why do we need it? git can tell you the last time it was changed

Reply:
iits useful for when we want to regenerate the image but we dont have any real changes to make to the file. The build wont catch it without some change

Reply:
do i need to add it manually?

Reply:
think youre right its probably not that important